Dockerfile

```diff
@@ -0,0 +1,57 @@
+# Build & test container
+FROM python:2.7-alpine as buildsys
+
+# Install required dependencies
+RUN apk add make
+RUN pip install pipenv
+
+# Add app user
+RUN adduser -D flaskd
+
+# Create app folder
+RUN mkdir -p /opt/om-parser-stw-potsdam-v2
+WORKDIR /opt/om-parser-stw-potsdam-v2
+
+# Copy app folder contents
+COPY stw_potsdam/ ./stw_potsdam
+COPY tests ./tests
+COPY Makefile .
+COPY Pipfile .
+COPY Pipfile.lock .
+
+# Apply app user to app folder
+RUN chown -R flaskd:flaskd .
+
+# Prepare environment
+USER flaskd
+ENV PYTHONPATH="/opt/om-parser-stw-potsdam-v2:${PYTHONPATH}"
+
+# Install environment
+RUN pipenv install --two --dev
+
+# Run tests
+RUN make test
+
+
+# Executable container
+
+FROM python:2.7-alpine
+
+RUN apk add --no-cache uwsgi uwsgi-python
+RUN pip install pipenv
+RUN adduser -D flaskd
+
+RUN mkdir -p /opt/om-parser-stw-potsdam-v2
+COPY --from=buildsys /opt/om-parser-stw-potsdam-v2 /opt/om-parser-stw-potsdam-v2
+
+WORKDIR /opt/om-parser-stw-potsdam-v2
+RUN chown -R flaskd:flaskd .
+
+USER flaskd
+ENV PYTHONPATH="/opt/om-parser-stw-potsdam-v2:${PYTHONPATH}"
```

Comment:
LGTM, works like a charm. In order to fully understand, just two minors:
* Why tinker with `PYTHONPATH`? I does not appear relevant to me, I suppose pipenv should take care of that.
* Maybe a (env) variable for the app path would be nice, instead of spelling it out at least five times.